connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -122,6 +123,8 @@ public class ErrorHandlingTaskTest {
 
     @Mock
     Plugins plugins;
+    @Mock
+    TransformationStage<?> transformation;
```

Comment:
```suggestion
    TransformationStage<?> transformationStage;
```

nit: since it's a little confusing to read `transformation.transformClass`